docs/Consumer/consumer_data_access_subscription.md

```diff
@@ -0,0 +1,89 @@
+---
+sidebar_position: 5
+---
+
+# Data Access through Subscription
+
+To register for a subscription, users must request consent from the AIP, obtain a [token](./consumer_obtaining_access_token.md) from the ADeX Auth server, and then access the [Subscription API](https://rs.ts.adex.org.in/apis#tag/Data-Subscriber).
+
+### Obtaining a Token to Subscribe to a Resource
+
+Users can obtain a token using the [Create Token APIs](https://ts.adex.org.in/auth/apis#tag/Token-APIs/operation/post-auth-v1-token) with the following request body. Ensure that the appropriate resource or resource group ID is used to obtain a token.
+
+*Example for a subscription at a resource level:*
+
+```json
+{
+  "itemId": "31b3da3a-84da-4923-b425-a9c5469c169f",
+  "itemType": "resource",
+  "role": "consumer"
+}
+```
+
+*Example for a subscription at a resource group level:*
+
+```json
+{
+  "itemId": "a007f760-8580-4401-81c1-b123da3de06f",
+  "itemType": "resource_group",
+  "role": "consumer"
+}
+```
+
+It should be noted that unless an explicit policy for subscription is specified by the AIP in the ADeX Authorization Server, users cannot obtain an access token for the secure resource.
```

Comment:
`...policy for subscription for the resource is specified by the AIP, users cannot...`